### ISSUE TITLE
transpose Coordinates with DependentCoordinates

### ISSUE THREAD

**Issue by jmilloy** [creare-com/podpac#254]
Coordinates with DependentCoordinates `lat,lon` can (now) be transposed. However we should be able to transpose the dependent dimensions, as well.

```
lat = np.linspace(0, 1, 12).reshape(3, 4)
lon = np.linspace(0, 1, 12).reshape(3, 4)
time = ['2018-01-01', '2018-010-2']
c = Coordinates([[lat, lon], time], dims=['lat,lon', 'time'])

# this works:
c.transpose('time', 'lat,lon')

# this does not
c.transpose('lon,lat', 'time')
```

**Comment by jmilloy** [comment]
There is a test failing for this. The fix is straightforward but not trivial.

**Comment by jmilloy** [comment]
The bug is fixed in 6794ddf, and the rest can wait.
